### PR TITLE
Added a field for sending JSON body with Rest Api Queries 

### LINKED ISSUE #2666: Feature: Add a field for sending JSON body with Rest Api Queries
### Is your proposal related to a problem?

Yes. Currently, We don't have an option to send JSON strings or objects along with Rest API queries  

### Describe the solution you'd like

Add a field for that. And also need a toggle to show json body field instead of the body form  

### If the feature is approved, would you be willing to submit a PR?
Yes

### PR DESCRIPTION
**Pull request by shah21**
Resolves #2666

### CODE REVIEW

File: frontend/src/Editor/QueryManager/QueryEditors/Restapi/TabBody.jsx [hunk 1]
```diff
@@ -9,21 +9,32 @@ export default ({
...
+  onJsonBodyChange,
   componentName,
+  bodyToggle,
+  setBodyToggle,
 }) => {
   return (
     <>
-      <GroupHeader addNewKeyValuePair={addNewKeyValuePair} paramType={'body'} descText={'Body Parameters'} />
-
+      <div style={{ display: 'flex' }}></div>
```
Comment (Navaneeth-pk) [inline frontend/src/Editor/QueryManager/QueryEditors/Restapi/TabBody.jsx:19]: Can we use the `d-flex` class instead of inline CSS?

**Comment by Navaneeth-pk** [comment]
If the body is a JSON array, it won't work.
Eg: `{{[1,2,3]}}`

Also, instead of trying to convert the JSON object to key value pairs, can we send the object as it is to the server? and based on `raw body` flag, the server can just use the object or proceed with existing flow?

@shah21

**Comment by Nandinisaha13** [comment]
<html>
<body>
<google-sheets-html-origin>Test Cases

Test Case | Steps | Result
-- | -- | --
Should be able to provide JSON body with Rest API queries. | Click on Datasource & select Rest API. | Selected Rest API
  | Enter the URL(pipedream) and click save. | Entered URL and saved it.
  | Toggle the json option in body, enter the body in json format. | The body should be visible in json format.
  | Click on preview. | Send the queries to pipedream.
  | View the ouput on piepedream, | It matches the input given above.
Should be able to authenticate with OAuth 2.0 | Click on Datasource & select Rest API. | Selected Rest API
  | Enter the details as per documentation & click on save. | The details should be entered & saved.
  | Create a query for get method & enter header click on preview. | It asks the account to get selected.
  | Select the account & close the window. | It shows the account detail in json.


</body>
</html>

**Review by akshaysasidrn — APPROVED** [review]
Nice! LGTM.

**Pull request merged**